app/com/gu/contentapi/sanity/package.scala

```diff
@@ -23,10 +23,11 @@ package object sanity extends ScalaFutures with Matchers with IntegrationPatienc
   def incrementIncidentCount = {
     incidentCount += 1
     lastIncidentDateTime = Some(DateTime.now)
+    println(s"Incident count: $incidentCount")
```

Comment:
It is a bit useful since I redirect `stdout` to a log file and I can go back and check how this is working but probably a proper logger would be better in that case. Any suggestions?